schema/public.openapi.yaml

```diff
@@ -205,6 +205,15 @@ components:
           $ref: '#/components/schemas/RunStatus'
         run:
           $ref: '#/components/schemas/Run'
+        links:
```

Comment:
@jharting Out of curiosity do you have plans to add filtering by inventory_host?

Reply:
Yes, that was my plan but I forgot to include it here. I am going to add that. Good catch!